pkg/controllers/api/api_controller.go

```diff
@@ -0,0 +1,25 @@
+package api
+
+import (
+	"fmt"
+	"github.com/3scale-labs/kamwiel/pkg/services"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+func Get(ctx *gin.Context) {
+	name := ctx.Param("name")
+	if len(name) == 0 {
+		ctx.JSON(http.StatusBadRequest, "Missing param `name`")
+		return
+	}
+	api, getErr := services.APIService.GetAPI(name)
+	if getErr != nil {
+		fmt.Println("API not found", getErr)
```

Comment:
you cannot be sure that the API is not found if err exists, can you?

Reply:
True that! I can, between many others, get client errors when connecting